.travis.yml

```diff
@@ -2,9 +2,12 @@ sudo: false
 
 language: go
 
+go_import_path: github.com/prometheus/graphite_exporter
+
 go:
-- 1.5.4
-- 1.6.2
+ - 1.7.x
```

Comment:
This should be consistent with Circle, and we only really need whatever we're currently building with.

Reply:
done